pkg/types/types.go

```diff
@@ -1431,6 +1431,9 @@ func NewDelimitedBlock(kind BlockKind, elements []interface{}, attributes interf
 		log.Debugf("overriding kind '%s' to '%s'", kind, k)
 		kind = k
 	}
+	if k, ok := attrs.GetAsString(AttrStyle); ok {
```

Comment:
This will be used as I convert the delimited blocks over to the new BlockAttrs syntax -- it should reduce the transition effort.